apps/api_banking/lib/api_banking/transactions/create_transfer.ex

```diff
@@ -0,0 +1,120 @@
+defmodule ApiBanking.Transactions.CreateTransfer do
+  alias ApiBanking.Repo
+  alias Ecto.Multi
+  alias ApiBanking.Transactions.Schemas.Transaction
+  alias ApiBanking.Accounts.Schemas.Account
+  import Ecto.Query, only: [where: 3, lock: 2]
+
+  require Logger
+
+  def create_transfer(struct) when is_struct(struct) do
```

Comment:
Aqui você podia fazer o pattern match direto com a struct de input que pretende receber, ex.:
```suggestion
  def create_transfer(%Transfer{} = input) do
```